src/Resources/doc/index.rst

```diff
@@ -0,0 +1,121 @@
+SymfonyMakerBundle
+==================
+
+Symfony Maker helps you creating empty commands, controllers, form classes,
+tests and more so you can forget about the required boilerplate code. This
+bundle is an alternative to `SensioGeneratorBundle`_ for modern Symfony
+applications and requires using Symfony 3.4 or newer and `Symfony Flex`_.
+
+Installation
+------------
+
+Run this command to install and enable this bundle in your application:
+
+.. code-block:: terminal
+
+    $ composer require maker
+
+Usage
+-----
+
+This bundle provides several commands under the ``maker:`` namespace. List them
+all executing this command:
+
+.. code-block:: terminal
+
+    $ php bin/console make:
+
+     make:command            Creates a new console command class
+     make:controller         Creates a new controller class
+     make:entity             Creates a new Doctrine entity class
+
+     [...]
```

Comment:
What about to explicitly list all the commands here? We'd know exactly all that is possible to _make_ only by reading the readme.

Reply:
I prefer to not do that to avoid having to maintain that list forever. This bundle just started, so we'll update this list of commands often ... and the less work we put on ourselves (e.g. maintaining this doc) the better.